modules/navier_stokes/test/tests/finite_volume/controls/switch-pressure-bc/test.i

```diff
@@ -0,0 +1,282 @@
+  []
+  [u_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_x
+    velocity_interp_method = ${velocity_interp_method}
+    advected_interp_method = ${advected_interp_method}
+    rho = ${rho}
+    momentum_component = 'x'
+  []
+  [u_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_x
+    mu = ${mu}
+    momentum_component = 'x'
+  []
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_x
+    momentum_component = 'x'
+    pressure = pressure
+  []
+
+  [v_time]
+    type = WCNSFVMomentumTimeDerivative
+    variable = vel_y
+    drho_dt = drho_dt
+    rho = rho
+    momentum_component = 'y'
+  []
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_y
+    velocity_interp_method = ${velocity_interp_method}
+    advected_interp_method = ${advected_interp_method}
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_y
+    momentum_component = 'y'
+    mu = ${mu}
+  []
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_y
+    momentum_component = 'y'
+    pressure = pressure
+  []
+
+  [temp_time]
+    type = WCNSFVEnergyTimeDerivative
+    variable = T_fluid
+    cp = cp
+    rho = rho
+    drho_dt = drho_dt
+  []
+  [temp_conduction]
+    type = FVDiffusion
+    coeff = 'k'
+    variable = T_fluid
+  []
+  [temp_advection]
+    type = INSFVEnergyAdvection
+    variable = T_fluid
+    velocity_interp_method = ${velocity_interp_method}
+    advected_interp_method = ${advected_interp_method}
+  []
+  [heat_source]
+    type = FVCoupledForce
+    variable = T_fluid
+    v = power_density
+  []
+[]
+
+[FVBCs]
+  [no_slip_x]
+    type = INSFVNoSlipWallBC
+    variable = vel_x
+    boundary = 'top bottom'
+    function = 0
+  []
+
+  [no_slip_y]
+    type = INSFVNoSlipWallBC
+    variable = vel_y
+    boundary = 'top bottom'
+    function = 0
+  []
+
+  # Inlet
+  [inlet_u]
+    type = INSFVInletVelocityBC
+    variable = vel_x
+    boundary = 'left'
+    function = ${inlet_v}
+  []
+  [inlet_v]
+    type = INSFVInletVelocityBC
+    variable = vel_y
+    boundary = 'left'
+    function = 0
+  []
+  [inlet_T]
+    type = FVDirichletBC
+    variable = T_fluid
+    boundary = 'left'
+    value = ${inlet_temp}
+  []
+
+  [outlet_p]
+    type = INSFVOutletPressureBC
+    variable = pressure
+    boundary = 'right'
+    function = ${outlet_pressure}
+  []
+  [outlet_p_later]
+    type = INSFVOutletPressureBC
+    variable = pressure
+    boundary = 'right'
+    function = ${fparse 2 * outlet_pressure}
+    enable = false
+  []
+[]
+
+[Functions]
+  [conditional_function]
+    type = ParsedFunction
+    expression = 't > 1.5'
+  []
+[]
+
+[Controls]
+  [p_threshold]
+    type = ConditionalFunctionEnableControl
+    conditional_function = conditional_function
+    disable_objects = 'FVBCs::outlet_p'
+    enable_objects = 'FVBCs::outlet_p_later'
+    execute_on = 'INITIAL TIMESTEP_BEGIN'
```

Comment:
would you mind switching the test to switching inlet and outlet?

that s the final use case anyway

Reply:
so outlet_p_later would live on left, and inlet_vel_later would live on right